### ISSUE TITLE
Wide Residual Networks

### ISSUE THREAD

**Issue by chullhwan-song** [chullhwan-song/Reading-Paper#88]
https://arxiv.org/abs/1605.07146

**Comment by chullhwan-song** [comment]
## Abstract
* Deep residual networks는 현재 수천개의 layer들을 쌓을수 있다. for performance..
   * 향상된 acc에 비해 layer는 거의 두배이상 쌓아야한다.
   * 학습에 feature의 재사용에 대한 문제가 있어서, 학습이 매우 오래걸림. 
* 이 연구에서는 이러한 문제를 해결하기 위해, depth 보다는 width한 residual network를 제한.
    * **wide residual networks (WRNs)**
    * 그래서, "16-layer-deep wide residual network" > 기존의 성능보다 우수하다는 걸 보임.
        * 다음 dataset에서, CIFAR, SVHN, COCO, and significant improvements on ImageNet. 

## contributions
* residual network architecture에 대한 상세한 연구
* **widened architecture** for ResNet blocks를 제시하여 성능 향상됨을 보임.
* deep residual network안에서 새로운 dropout 를 사용방법을 제시 > regularize, overfitting 
* SOTA 성능

## Wide residual networks
![image](https://user-images.githubusercontent.com/40360823/53560436-ad938880-3b8f-11e9-9850-89df6cd9a973.png)
* Residual block with identity mapping
![image](https://user-images.githubusercontent.com/40360823/53560087-fb5bc100-3b8e-11e9-8b25-24772bf64175.png)
   * F는 residual network
      * ![image](https://user-images.githubusercontent.com/40360823/53560208-35c55e00-3b8f-11e9-8c67-eb07fd4bef33.png)  net의 parameter
   * l는 l번째 layer에서의 unit(feature)
* 기존 연구의 residual networks  - 2개의 type 
   * basic - conv3×3-conv3×3, Fig1 a)
      * conv3×3은 batch normalization & ReLU 포함.
   * bottleneck - conv1×1-conv3×3-conv1×1, Fig1 b)
